### ISSUE TITLE
Octave does not use KDE open/save dialogs

### ISSUE THREAD

**Issue by kupiqu** [flathub/org.octave.Octave#28]
Being a Qt app and depending on KDE, I expect Octave to use KDE open/save dialogs. This is not what currently happens.

Now I don't know it this is or not related to issue #6 regarding the recent change in flatpak. However, KDE apps, e.g. Kdenlive, show the native KDE open/save dialogs.

**Comment by kupiqu** [comment]
It's interesting because calling uigetfile in the console uses the KDE dialog, so I don't think it's related to issue #6

**Comment by mtmiller** [comment]
I remembered some mention about native dialogs a while ago, and I found [this bug report](https://savannah.gnu.org/bugs/?38305) and [this change](https://hg.savannah.gnu.org/hgweb/octave/rev/6bd74153c3ae).

tl;dr - KDE native file dialogs use QProcess, and QProcess can't be used with Octave. So it's deliberate, and has nothing to do with flatpak.

**Comment by kupiqu** [comment]
Thanks for the info @mtmiller  but this was Qt4 bug, which is not used in current Octave versions. As mentioned above the `uigetfile` command opens a native KDE dialog and nothing conflicts/breaks.
 
I agree this discussion does not correspond here, but could we reopen the bug in https://savannah.gnu.org/bugs/?38305 to revisit the changes in https://hg.savannah.gnu.org/hgweb/octave/rev/6bd74153c3ae 

Unless I am missing sth, it seems the conflict doesn't exist anymore...

**Comment by mtmiller** [comment]
Sure, but please open a new bug, and just link to the older bug and commit for context.

**Comment by kupiqu** [comment]
Done: https://savannah.gnu.org/bugs/index.php?54607
